composer.json

```diff
@@ -6,7 +6,7 @@
         "ext-ctype": "*",
         "ext-iconv": "*",
         "ext-sodium": "*",
-        "cidilabs/phpally": "^1.0.2",
+        "cidilabs/phpally": "^1.0.5",
```

Comment:
Change to `~1.1.0`.  That version will have the link checker test re-enabled and include other changes.